src/os/bluestore/BitAllocator.cc

```diff
@@ -1669,3 +1707,11 @@ void BitAllocator::free_blocks_dis(int64_t num_blocks, ExtentList *block_list)
   alloc_assert(get_used_blocks() >= 0);
   unlock();
 }
+
+void BitAllocator::dump()
+{
+  int count = 0;
+  serial_lock(); 
+  dump_int(count);
```

Comment:
we can get the dump logic here, instead of adding another no lock function. Dump is just some debug functionality, no need to have different versions.